test/functional/hostgroup/info_test.rb

```diff
@@ -0,0 +1,32 @@
+require_relative '../test_helper'
+require 'hammer_cli_foreman/hostgroup'
+
+module HammerCLIForeman
+  describe Hostgroup do
+    describe InfoCommand do
+      before do
+        @cmd = %w(hostgroup info)
+      end
+
+      let(:hostgroup_id) { 1 }
+
+      it "Shows information about a hostgroup" do
+        params = ["--id=#{hostgroup_id}"]
+
+        ex = api_expects(:hostgroups, :show, "Get info") do |par|
+          par["id"] == hostgroup_id.to_s
+        end
+        json_file = File.join(File.dirname(__FILE__), 'data', 'hostgroup.json')
+        ex.returns(JSON.parse(File.read(json_file)))
+
+        result = run_cmd(@cmd + params)
+        expected_fields = [['Lifecycle Environment', 'Library'],
+                           ['Content View', 'Default Organization View'],
+                           ['Environment', 'Production Env'],
+                           ['Name', 'my host group']]
```

Comment:
Thank you for writing a test! One small thing - let's remove the checks for `'Name'` and `'Environment'`, since those fields are coming from hammer-cli-foreman. If you want to test them, submit a PR to hammer-cli-foreman with a hostgroups#list_command test just like this one, just removing the `'Lifecycle Environment'` and `'Content View'` fields.